content/middlewares.md

```diff
@@ -243,4 +243,14 @@ app.use(logger);
 await app.listen(3000);
 ```
 
+#### Default body-praser middleware
+The nest application registers `json` and `urlencoded` from the package `body-parser` by default. This can be turned off by setting the `bodyParser` flag when creating the application. The options used when registering the middleware can be seen in the `registerParserMiddleware` function in the (express)[https://github.com/nestjs/nest/blob/master/packages/platform-express/adapters/express-adapter.ts#L210] and (fastify)[https://github.com/nestjs/nest/blob/master/packages/platform-fastify/adapters/fastify-adapter.ts#L455] adapters.
```

Comment:
```suggestion
#### Default body parser middleware

When using the `express` adapter, the NestJS app will register `json` and `urlencoded` from the package `body-parser` by default. This can be turned off by setting the `bodyParser` flag to `false` when creating the application. The options used when registering the middleware can be seen in the `registerParserMiddleware` function in the (express)[https://github.com/nestjs/nest/blob/master/packages/platform-express/adapters/express-adapter.ts#L210] and (fastify)[https://github.com/nestjs/nest/blob/master/packages/platform-fastify/adapters/fastify-adapter.ts#L455] adapters.
```